### DISCUSSION TITLE
Neos Cam Support? Wyze Clone Cameras

### DISCUSSION THREAD

**Discussion by evenwebb** [mrlt8/docker-wyze-bridge#239] [Ideas]
Hi all

This software looks great and thank you for spending the time to create it.

I was wondering if support for the Neos Smart Cams is something that could be added? 

They are the same hardware just a different app and backend.

Cheers

**Comment by mrlt8** [comment]
Possibly. We'd have to dig into the Neos app to see how the app actually connects to the camera.

**Comment by evenwebb** [comment]
Perhaps you might consider it if others are interested and you have time :)

Thank you
